lib/calendarium-romanum/util.rb

```diff
@@ -23,6 +23,7 @@ def enumeration_over?(date)
 
     # enumerates days of a year
     class Year < DateEnumerator
+      attr_reader :start
```

Comment:
It seems `Year#start` and `Month#start` are not used anywhere. No reason to add an attribute.

Reply:
Done. Just posting it here as a checklist for myself.